contracts/BalanceTrackerFixedPriceNative.sol

```diff
@@ -37,11 +37,8 @@ contract BalanceTrackerFixedPriceNative is BalanceTrackerFixedPriceBase {
 
     function _checkNativeValue() internal virtual override {}
 
-    function _getRequiredFunds(address, uint256 balanceDiff) internal virtual override returns (uint256) {
-        if (msg.value < balanceDiff) {
-            revert InsufficientBalance(msg.value, balanceDiff);
-        }
-        return msg.value;
+    function _getRequiredFunds(address, uint256) internal virtual override returns (uint256) {
+        return 0;
```

Comment:
Now it matches the method as well (in the case of native token there is no get, just push)